packages/bootleg-tokens/contracts/ISharedRoyaltyToken.sol

```diff
@@ -0,0 +1,64 @@
+pragma solidity ^0.5.3;
+
+import "openzeppelin-solidity/contracts/token/ERC721/IERC721.sol";
+
+/**
+ * @title Shared Royalty Non-Fungible Token Standard basic interface
+ */
+contract ISharedRoyaltyToken is IERC721 {
+
+  /**
+   * @notice Gets the franchisor balance of the specified address
+   *
+   * @param _franchisor address to query the balance of
+   * @return uint256 representing the amount franchised by the passed address
+   */
+  function franchisorBalanceOf(address _franchisor) public view returns (uint256 balance);
+
+  /**
+   * @notice Withdraws payment accumulated from transfer of a given token from
+   *  the last withdrawn payment up to a _count
+   *
+   * @param _tokenId The identifier for an NFT
+   * @param _count The number of payments to traverse
+   */
+  function withdrawPayment(uint256 _tokenId, uint256 _count) public;
+
+  /**
+   * @notice Withdraws non-withdrawn payment accumulated from transfer of a
+   *   given token
+   * @dev Escrow should keep track of the payments that have been processed to
+   *   avoid double withdrawals
+   *
+   * @param _tokenId The identifier for an NFT
+   */
+  function withdrawPayment(uint256 _tokenId) public;
+
+  /**
+   * @notice Gets balance of non-withdrawn payment accumulated from transfer of a
```

Comment:
I'm seeing this 'non-withdrawn' word a lot and thinking it's an important concept I'm missing.

Reply:
Tried to explain above, hopefully this comment is helpful https://github.com/ConsenSys/web3studio-bootleg/issues/90#issuecomment-463299189